.github/workflows/e2e.gradle.workflow_dispatch.main.default.slsa3.yml

```diff
@@ -0,0 +1,107 @@
+on:
+  schedule:
+    - cron: "0 6 * * *"
+  workflow_dispatch:
+    inputs:
+      trigger_build:
+        description: "internal: do not check"
+        required: false
+        default: false
+        type: boolean
+
+permissions: read-all
+
+concurrency: "e2e-gradle-workflow_dispatch-main-default-slsa3"
+
+env:
+  # TODO(#263): create dedicated token
+  GH_TOKEN: ${{ github.token }}
```

Comment:
re-use the nodejs or go PAT token. Thera are problems unless we use a PAT (it's a GitHub thing)